### PR TITLE
OCM-8414 | ci: Fix 67576 of shared vpc profile creation

### PR DESCRIPTION
**Pull request by jameszwang**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story (OCM-xxxx)*:
Fixes #[OCM-8414](https://issues.redhat.com//browse/OCM-8414)

**Change type**
- [ ] New feature
- [ ] Bug fix
- [ ] Build
- [ x] CI
- [ ] Documentation
- [ ] Performance
- [ ] Refactor
- [ ] Style
- [ ] Unit tests
- [ ] Subsystem tests

**Checklist**
- [ x] Subject and description added to both, commit and PR.
- [ x] Relevant issues have been referenced.

### CODE REVIEW

File: tests/ci/profile_handler.go [hunk 1]
```diff
@@ -421,7 +421,7 @@ func GenerateClusterCreationArgsByProfile(token string, profile *Profile) (clust
 			//		* The Shared-VPC compatible policie requries installer role
 			//		* The install role (account roles) require Shared-VPC ARN.
 			//  Use hardcode as a temporary solution.
-			shared_vpc_role_arn = fmt.Sprintf("arn:aws:iam::641733028092:role/%s-shared-vpc-role", clusterArgs.ClusterName)
+			shared_vpc_role_arn = fmt.Sprintf("arn:aws:iam::641733028092:role/%s-shared-vpc-role", *clusterArgs.ClusterName)
```
Comment (xueli181114) [inline tests/ci/profile_handler.go:424]: Can you please create a jira ticket to track the enhancement of this part rather than hardcode the role arn here?

Comment (jameszwang) [reply tests/ci/profile_handler.go:424]: create a JIRA card(OCM-8543) to track this.

**Pull request merged**